Podfile.lock

```diff
@@ -1,11 +1,11 @@
 PODS:
```

Comment:
Can we add `Podfile.lock` and `Pods folder` to .gitignore as well? 😅  What do you think?

Reply:
yeah sure, I think it is good too. let me update 👍